webpack.config.js

```diff
@@ -18,37 +18,45 @@ const { getIfUtils, removeEmpty } = require( 'webpack-config-utils' );
 
 
 module.exports = ( env ) => {
-  const { ifProd, ifNotProd, ifTest, ifNotTest } = getIfUtils( env );
+  const { ifProd, ifNotProd, ifTest, ifDev, ifSite } = getIfUtils( env, ['prod', 'test', 'dev', 'site'] );
+
+  function _packagePrefix() {
+    return './packages' + ( env.element ? `/${env.element}` : '' );
+  }
+
+  function _entryPoint() {
+    if ( ifTest() ) {
+
+      return {
+        'test': _packagePrefix() + '/index.test.ts',
+        'test-helpers' : './test-helpers.ts'
+      };
+
+    }
+
+    if ( ifProd() ) {
+
+      return {
+        'index': _packagePrefix() + '/index.ts',
```

Comment:
well, because the config is wrong written. Next time don't hesitate to discuss things with me. @elmariofredo is not the only devstack "guru" in the room ;) :P Anyway last 2 devstacks that are open source were build primarily by me :P just saying ☮

here is what we need to do:
https://webpack.js.org/concepts/configuration/#multiple-targets